boring-sys/Cargo.toml

```diff
@@ -81,3 +81,6 @@ bindgen = { workspace = true }
 cmake = { workspace = true }
 fs_extra = { workspace = true }
 fslock = { workspace = true }
+
+[lints.rust]
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(const_fn)'] }
```

Comment:
I ended up just removing the `cfg(const_fn)`s, they seemed like a relic from when const wasn't stable